ZTNA APIs [Public Preview] (#21578)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/networkaccess-crosstenantaccesssettings.md

---
title: "crossTenantAccessSettings resource type"

> description: "A tenant's cross tenant access settings define Universal tenant restrictions version 2. The universal tenant restrictions version 2 offer coverage for tagging all traffic regardless of the browser or operating system used on the device, support for branch connectivity and direct from device connectivity, and native integration with Microsoft 365 and Azure Active Directory."

Reviewer: Limited field length; In Graph we don't have a concept of different TR versions.
```suggestion
description: "A tenant's cross tenant access settings defines universal tenant restrictions."
```

author: "Moti-ba"
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: resourcePageType
---

# crossTenantAccessSettings resource type

Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> A tenant's cross tenant access settings define Universal tenant restrictions version 2. The universal tenant restrictions version 2 offer coverage for tagging all traffic regardless of the browser or operating system used on the device, support for branch connectivity and direct from device connectivity, and native integration with Microsoft 365 and Azure Active Directory.

Reviewer: ```suggestion
A tenant's cross tenant access settings defines universal tenant restrictions. The universal tenant restrictions offer coverage for tagging all traffic regardless of the browser or operating system used on the device, support for branch connectivity and direct from device connectivity, and native integration with Microsoft 365 and Azure Active Directory.
```

> For more information about cross-tenant access settings, see [Universal tenant restrictions](https://github.com/azure/global-secure-access/how-to-universal-tenant-restrictions)."

Reviewer: ```suggestion
For more information about cross-tenant access settings, see [Universal tenant restrictions](https://github.com/azure/global-secure-access/how-to-universal-tenant-restrictions).
```

Inherits from [microsoft.graph.entity](../resources/entity.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Get crossTenantAccessSettings](../api/networkaccess-crosstenantaccesssettings-get.md)|[microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md)|Read the properties and relationships of a [microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md) object.|
|[Update crossTenantAccessSettings](../api/networkaccess-crosstenantaccesssettings-update.md)|[microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md)|Update the properties of a [microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|id Inherited from [microsoft.graph.entity](../resources/entity.md).|
> |networkPacketTaggingStatus|microsoft.graph.networkaccess.status|Determines if the service inserts a header with the user tenant ID into network traffic	.The possible values are: `enabled`, `disabled`.|

Reviewer: ```suggestion
|id|String|Identifier. Inherited from [microsoft.graph.entity](../resources/entity.md).|
|networkPacketTaggingStatus|microsoft.graph.networkaccess.status|Determines if the service inserts a header with the user tenant ID into network traffic. The possible values are: `enabled`, `disabled`.|
```

---
title: "crossTenantAccessSettings resource type"

> description: "Cross tenant access settings. For now it only contains the Network Packet Tagging Status. The network packet tagging with the user tenant ID will eventually be destined to Azure AD, and it will enforce Tenant Restrictions Policies, known as TRv2 Policies, which prevents users from exfiltrating data to tenants outside their organization"

Reviewer: ```suggestion
description: " Defines whether the tenant restrictions in cross-tenant access settings are enabled for traffic profiles. Each tenant has only one cross-tenant access settings object."
```

author: "Moti-ba"
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: resourcePageType
---

# crossTenantAccessSettings resource type

Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Cross tenant access settings. For now it only contains the Network Packet Tagging Status. The network packet tagging with the user tenant ID will eventually be destined to Azure AD, and it will enforce Tenant Restrictions Policies, known as TRv2 Policies, which prevents users from exfiltrating data to tenants outside their organization

Reviewer 2: This reads like it came from an internal spec.  Please make this clearer for our developer audience

Reviewer: `+1.` Same sentiment as the conditionalaccesssettings description. And then as a last sentence, refer readers to /azure/global-secure-access/how-to-universal-tenant-restrictions for more information. "_For more information about cross-tenant access settings, see [Universal tenant restrictions](/azure/global-secure-access/how-to-universal-tenant-restrictions)._"

|[Create crossTenantAccessSettings](../api/networkaccess-settings-post-crosstenantaccess.md)|[microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md)|Create a new [microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md) object.|
|[Get crossTenantAccessSettings](../api/networkaccess-crosstenantaccesssettings-get.md)|[microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md)|Read the properties and relationships of a [microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md) object.|
|[Update crossTenantAccessSettings](../api/networkaccess-crosstenantaccesssettings-update.md)|[microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md)|Update the properties of a [microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md) object.|

> |[Delete crossTenantAccessSettings](../api/networkaccess-settings-delete-crosstenantaccess.md)|None|Delete a [microsoft.graph.networkaccess.crossTenantAccessSettings](../resources/networkaccess-crosstenantaccesssettings.md) object.|

Reviewer: Only GET and UPDATE are supported.

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|id Inherited from [microsoft.graph.entity](../resources/entity.md).|
> |networkPacketTaggingStatus|microsoft.graph.networkaccess.status|Determines if the service inserts a header with the user tenant ID into network traffic	.The possible values are: `enabled`, `disabled`.|

Reviewer: Suggestion
```suggestion
|id|String|Identifier for the setting. Inherited from [microsoft.graph.entity](../resources/entity.md).|
|networkPacketTaggingStatus|microsoft.graph.networkaccess.status| Determines whether tenant restrictions for cross-tenant access settings are enabled. The possible values are: `enabled`, `disabled`. <br/> When enabled, the service inserts a header that contains the user tenant ID and the tenant restrictions policy ID into network traffic. |
```
